### DISCUSSION TITLE
Subgraphs

### DISCUSSION THREAD

**Discussion by IbrahimIjai** [scaffold-eth/scaffold-eth-2#614] [Q&A]
I was thinking, when are we adding subgraphs to the starter kit??

Is it currently being worked on? The initial version of se has a subgraph.

If it is not currently being worked on, can you make it open to builders to work on it?

I also recently migrated to next14, everythings is currently working fine: https://github.com/IbrahimIjai/Scaffoldeth-with-next14

Lets make SE2 the best

**Comment by damianmarti** [comment]
Hi!

You can use this branch https://github.com/scaffold-eth/scaffold-eth-2/tree/subgraph-package

Subgraphs will be added soon using the extension system. In the meantime, you can use that branch.
